File: wwwroot/app/models/BugOrFeature.ts

export class BugOrFeature
{
    isbug : number;
    isfeature : number; 
    text : string; 

    constructor(isbug : number, isfeature : number, text : string)
    {
        this.isbug = isbug;
        this.isfeature = isfeature; 
        this.text = text; 
    }
}

File: Models/app/BugOrFeature.cs
using System;
using Newtonsoft.Json; 
using NPoco; 

namespace WebApplication.Models.app
{
    [JsonObject(MemberSerialization.OptOut)]
    [TableName("bugorfeature")]
    [PrimaryKey("id" , AutoIncrement=true)]   
    [ExplicitColumns]  
    public class BugOrFeature
    {
        [Column]
        public int ID;
        [Column]
        public string text;
        [Column]
        public int isbug;
        [Column]
        public int isfeature; 

    }
}